e2e/interchaintestv8/sp1_ics07_test.go

```diff
@@ -34,6 +38,9 @@ import (
 type SP1ICS07TendermintTestSuite struct {
 	e2esuite.TestSuite
 
+	// Whether to generate fixtures for the solidity tests
+	generateFixtures bool
```

Comment:
I added this to be able to generate fixtures from the e2e tests for misbehaviour. We had already talked about potentially doing this for all fixtures, so we don't depend on getting state from the mocha testnet as we do now. Instead we could generate them from the e2e tests which should be perfectly valid. We already do this for benchmarks in the solidity-ibc-eureka repo.